CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased (develop)
 
+- added: Scam warning to camera access request modal
```

Comment:
Nitpick: this should be at the bottom of the added list or even the changed list. There are other more interesting features that should be listed above this one.